### ISSUE TITLE
Getting error "urn:acme:error:malformed"

### ISSUE THREAD

**Issue by amitguptagwl** [Neilpang/acme.sh#1970]
## Installation
```bash
$ ./acme.sh --install
[DATE] It is recommended to install socat first.
[DATE] We use socat for standalone server if you use standalone mode.
[DATE] If you don't use standalone mode, just ignore this warning.
:
```

## Issue certificate
After restarting the terminal, I was trying to generate the certificate using following command after TXT entry;

```bash
$ acme.sh --renew --dns -d mydomani.com --yes-I-know-dns-manual-mode-enough-go-ahead-please
```
```json
 {"type":"urn:acme:error:malformed","detail":"Unable to update challenge :: authorization must be pending","status": 400}
```

**Comment by FernandoMiguel** [comment]
If you are running out of aws, why not use route 53 api instead of standalone?
Also keep in mind, ec2-user won't be able to reload services

**Comment by amitguptagwl** [comment]
Thanks. My original problem has been solved. 

Some DNS host provider needs the hostname as `subdomain.domain.tld` while other needs only `subdomain` when we create a TXT record. And if we've already run the command with --revert command, we can't re-run it.  So I've corrected the TXT entry then re-run both commands in sequence.

> If you are running out of aws, why not use route 53 api instead of standalone?

Sorry I didn't understand.

> Also keep in mind, ec2-user won't be able to reload services.

Hmm. I'm issuing the certificate manually. But need to automate it.
